### PR TITLE
Update MINISEED_METADATA_CONVENTIONS.md

### PR DESCRIPTION
**Pull request by staylorofford**

### CODE REVIEW

**Review by salichon — DISMISSED** [review]
it sounds good to me and 
I Reckon  this below is still valid @staylorofford   wrt our recent borehole discussion 
_"- Changes in position in a borehole are not considered as position changes, e.g., WHSZ borehole sensor moving from 5 m to 500 m would not require a change in the sensor's station or location code."_

Perhaps for later  _[0-2-5] meters_ depth instruments to be defined in addition  as "shallow/surface/(~bore~)hole"seismometers

**Review by ozym — APPROVED** [review]

**Comment by staylorofford** [comment]
Hi @salichon and @ozym, could I please get another review on these now that they are updated, then I can merge and we can start actioning them. COVZ is on my radar.

**Comment by staylorofford** [comment]
bump

File: docs/MINISEED_METADATA_CONVENTIONS.md
Comment (salichon) [inline docs/MINISEED_METADATA_CONVENTIONS.md:1]: conventions are very useful based upon this IRIS 
 link https://ds.iris.edu/ds/nodes/dmc/data/formats/seed-channel-naming/

**Review by salichon — APPROVED** [review]

**Pull request merged**